src/main/java/org/spongepowered/api/item/inventory/custom/InventoryArchetypes.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.item.inventory.custom;
+
+import com.flowpowered.math.vector.Vector2i;
+import org.spongepowered.api.data.Property;
+import org.spongepowered.api.item.inventory.InventoryProperty;
+import org.spongepowered.api.item.inventory.property.AbstractInventoryProperty;
+import org.spongepowered.api.item.inventory.property.InventorySize;
+import org.spongepowered.api.text.Text;
+
+/**
+ * An enumeration of some usefull Inventory {@link InventoryArchetype}s.
+ */
+public class InventoryArchetypes {
+
+    // Vanilla Archetypes
+    // TODO Dummies
+    public static final InventoryArchetype CHEST = null;
+    public static final InventoryArchetype WORKBENCH = null;
+
+    public static final InventoryArchetype MENU_ROW = null;
+    public static final InventoryArchetype MENU_GRID = null;
+
+    public static class Properties {
+
+        public static final InventoryProperty DOUBLE_CHEST = null;
+
+        public static InventorySize size(Vector2i dimension) {
```

Comment:
`int, int` method as well, `Vector2i` method can be a default which calls `int, int`